standalone/src/chain_spec.rs

```diff
@@ -54,12 +52,13 @@ fn get_properties() -> Map<String, Value> {
     let mut properties = Map::new();
     let mut token_symbol: Vec<String> = vec![];
     let mut token_decimals: Vec<u32> = vec![];
-    TokenSymbol::get_info().iter().for_each(|(symbol_name, decimals)| {
-        token_symbol.push(symbol_name.to_string());
-        token_decimals.push(*decimals);
+    [INTR, INTERBTC, DOT, KINT, KBTC, KSM].iter().for_each(|token| {
```

Comment:
If you are going to manually list them here, you might as well leave out the currencies that we don't use